libportability-gfx/src/impls.rs

```diff
@@ -399,9 +478,11 @@ pub extern "C" fn gfxGetInstanceProcAddr(
         vkEnumerateDeviceLayerProperties, PFN_vkEnumerateDeviceLayerProperties => gfxEnumerateDeviceLayerProperties,
 
         vkGetPhysicalDeviceFeatures, PFN_vkGetPhysicalDeviceFeatures => gfxGetPhysicalDeviceFeatures,
+        vkGetPhysicalDeviceFeatures2KHR, PFN_vkGetPhysicalDeviceFeatures2KHR => gfxGetPhysicalDeviceFeatures2KHR,
```

Comment:
How do we plan to handle the extension names in general once they're included in the core API (for example, when it's just `vkGetPhysicalDeviceFeatures2` if the version number is high enough)?

Reply:
we'll just map it to the same entry point? I'm not sure I understand the problem that would appear if we rise up the core API version